### ISSUE TITLE
systemtap-sdt is missing in install-dependencies script

### ISSUE THREAD

**Issue by data2** [scylladb/seastar#231]
Hi,

while trying to compile seastar, I had the below error

```
 core/reactor.cc:85:21: fatal error: sys/sdt.h: No such file or directory
 #include <sys/sdt.h>
                     ^
```

This was easily fixed by installing systemtap-sdt-dev on Ubuntu 16.10, but I guess that should be included in the install-dependencies.sh script

**Comment by hodgesds** [comment]
I experienced a similar problem with fedora 25:

```sh
FAILED: build/release/core/reactor.o 
g++ -MD -MT build/release/core/reactor.o -MF build/release/core/reactor.o.d -O2 -I build/release/gen -I build/release/c-ares -std=gnu++1y -g  -Wall -Werror -fvisibility=hidden -pthread -I. -U
_FORTIFY_SOURCE  -march=nehalem -Ifmt -DBOOST_TEST_DYN_LINK -Wno-overloaded-virtual -DFMT_HEADER_ONLY -DHAVE_HWLOC -DHAVE_NUMA -DHAVE_LZ4_COMPRESS_DEFAULT -c -o build/release/core/reactor.o c
ore/reactor.cc
core/reactor.cc:86:21: fatal error: sys/sdt.h: No such file or directory
 #include <sys/sdt.h>
```

and this seem to do the trick:
```sh
sudo dnf install systemtap-devel-3.1-0.20170125gite81970274b46.fc25.x86_64  systemtap-sdt-devel.x86_64 3.1-0.20170125gite81970274b46.fc25
```

**Comment by avikivity** [comment]
Fixed by 8b1eecd603d2ebdd2409b06e6fb0770b6c16c849, 3fd0a0efdf381d1b4e144f02160e78df98188be6, b5c3ac03fd40765349b389b9d3e34897a18a271e.
